libp2p/protocols/connectivity/relay/relay.nim

```diff
@@ -403,7 +406,7 @@ proc deletesReservation(r: Relay) {.async: (raises: [CancelledError]).} =
         if n > r.rsvp[k]:
           r.rsvp.del(k)
     except KeyError:
-      raiseAssert "checked with in"
+      raiseAssert "checked with in: " & getCurrentExceptionMsg()
```

Comment:
`raiseAssert "checked with"` constructs must never happen, they are there just to make compiler happy. there is no need in adding message here.